include/mbgl/map/query.hpp

```diff
@@ -8,13 +8,20 @@
 
 namespace mbgl {
 
+using StringVector = std::vector<std::string>;
```

Comment:
That's just an shorter alias - we adopt this idiom in some other parts of the codebase - but I'm :+1: in inlining it.